django/apps/registry.py

```diff
@@ -255,6 +255,17 @@ def get_registered_model(self, app_label, model_name):
                 "Model '%s.%s' not registered." % (app_label, model_name))
         return model
 
+    @lru_cache.lru_cache(maxsize=None)
```

Comment:
A comment explaining why caching matters would help future maintainers. See the comment above `get_models` for an example.